src/model.md

```diff
@@ -0,0 +1,171 @@
+# vscode-code-review Model Usage
+
+- [How To Add A Property](#how-to-add-a-property)
+- [How To Delete A Property](#how-to-delete-a-property)
+
+## How To Add A Property
+
+In this sample, we are going to add a new property `state` of type `number`.  
+All the steps will be operated in the file `model.ts`.
+
+1. Add the property to the **interface** `CsvEntry`:
+
+    Before:
+
+    ```typescript
+    export interface CsvEntry {
+        sha: string;
+        filename: string;
+        url: string;
+        lines: string;
+        title: string;
+        comment: string;
+        priority: number;
+        category: string;
+        additional: string;
+        code?: string;
+        /** Unique identifier of the entry */
+        id: string;
+    }
+    ```
+
+    After:
+
+    ```typescript
+    export interface CsvEntry {
+        sha: string;
+        filename: string;
+        url: string;
+        lines: string;
+        title: string;
+        comment: string;
+        priority: number;
+        category: string;
+        additional: string;
+        code?: string;
+        /** Unique identifier of the entry */
+        id: string;
+        /** The new state property */
+        state: number;
+    }
+    ```
```

Comment:
```suggestion

    ```diff
    export interface CsvEntry {
        sha: string;
        filename: string;
        url: string;
        lines: string;
        title: string;
        comment: string;
        priority: number;
        category: string;
        additional: string;
        code?: string;
        /** Unique identifier of the entry */
        id: string;
+       /** The new state property */
+       state: number;
    }
    ```
```